plutus-pab/src/Plutus/PAB/Core/ContractInstance.hs

```diff
@@ -193,6 +193,8 @@ stmRequestHandler = fmap sequence (wrapHandler (fmap pure nonBlockingRequests) <
         <> RequestHandler.handleOwnInstanceIdQueries @effs
         <> RequestHandler.handleAddressChangedAtQueries @effs
         <> RequestHandler.handleCurrentSlotQueries @effs
+        <> RequestHandler.handleTimeNotifications @effs
```

Comment:
This handler answers `awaitTime` requests, so it's a blocking request

Reply:
Good catch! should be good now